kolibri/plugins/learn/assets/src/modules/pluginModule.js

```diff
@@ -20,6 +20,7 @@ export default {
     examLog: {},
     memberships: [],
     canAccessUnassignedContentSetting: plugin_data.allowLearnerUnassignedResourceAccess,
+    allowGuestAccess: plugin_data.allowGuestAccess,
```

Comment:
That's weird - we were already bootstrapping this into the learn page, but somehow we don't seem to be using it anywhere before this PR?

Reply:
Yeah I thought that was odd, but appreciated it being there :-D